hummingbird/forms.py

```diff
@@ -0,0 +1,33 @@
+from django import forms
+from django.contrib.auth.models import User
+from models import UserProfile, UserDevice
```

Comment:
nit: Newline between library imports (django) and local imports (models module).